test/unit/definitions/divorce/CaseTypeTab.test.js

```diff
@@ -92,6 +92,7 @@ describe('CaseTypeTab', () => {
     serviceApplication: 23,
     LinkedCase: 24,
     Language: 25,
+    ConfidentialDocuments: 26,
```

Comment:
I don't know why it's here. It was a part of:
https://github.com/hmcts/div-ccd-definitions/pull/423

already merged into master. Should not be displayed here.